fastlane/fastlane
# [fastlane_core][match] improve copy when asking user for password in interactive mode

### Checklist

### Motivation and Context
The current copy is confusing because it asks for `password for X` and `confirm passphrase for Y` 😅 

### Description

Recently I was running _match_ and I was asked the "passphrase" earlier in the flow, and then when it prompted for my keychain password, this happened:

```
[10:28:54]: Installing certificate...
[10:28:56]: Enter the password for /Users/rogerluan/Library/Keychains/login.keychain-db
[10:28:56]: This passphrase will be stored in your local keychain with the name fastlane_keychain_login and used in future runs
[10:28:56]: This prompt can be avoided by specifying the 'keychain_password' option or 'MATCH_KEYCHAIN_PASSWORD' environment variable
[10:28:56]: Password for login keychain: **************
[10:29:08]: Type passphrase again: **************************************************
[10:29:18]: Passphrases differ. Try again
[10:29:18]: Password for login keychain: **************
[10:29:23]: Type passphrase again: **************
[✔] Setting key partition list... (this can take a minute if there are a lot of keys installed) 
```

The `type passphrase again` was confusing since the previous prompt was `password for login keychain`. This PR fixes that:

```
Password for login keychain: **************
Type password for login keychain again: **************
```

### Testing Steps
To test this branch, modify your Gemfile as:

```ruby
gem "fastlane", :git => "https://github.com/fastlane/fastlane.git", :branch => "rogerluan-improve-password-helper-copy"
```

And run `bundle install` to apply the changes. 

To check the issue, run match in interactive mode.
